app/ollama.iss

```diff
@@ -60,7 +60,7 @@ MinVersion=10.0.10240
 
 ; quiet...
 DisableDirPage=yes
-;DisableFinishedPage=yes
+DisableFinishedPage=yes
```

Comment:
Stretch goals for later:

1. Remove first page
2. Screen with progress bar has lower height for a more minimal design